defaults/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
 # Install from phar (faster, but less flexible).
 drush_version: "8.1.10"
```

Comment:
Should probably remove this setting as well, as you suggested in #41. +1 otherwise though.

@geerlingguy Drush 9 works with 8.3+, and 8.4 is out now. Drupal VM will also need this in order to be able to install Drush 9.

Reply:
IMO, this should stay.

[Drush 8.x docs](http://docs.drush.org/en/8.x/install/) still instruct to download the `drush` phar instead of `drush-launcher`, and the role should follow the docs for installing `8.x`.